From stable-bounces@linux.kernel.org Fri Feb  1 15:31:02 2008
From: Stefan Bader <stefan.bader@canonical.com>
Date: Fri,  1 Feb 2008 15:18:38 -0800
Subject: USB: Fix usb_serial_driver structure for Kobil cardreader driver.
To: linux-usb@vger.kernel.org
Cc: Alan Cox <alan@redhat.com>, Greg Kroah-Hartman <gregkh@suse.de>, stable <stable@kernel.org>, Stefan Bader <stefan.bader@canonical.com>
Message-ID: <1201907929-23721-117-git-send-email-gregkh@suse.de>


From: Stefan Bader <stefan.bader@canonical.com>

The device setup did miss to initialize the num_interrupt_out field, thus
failing to successfully complete the probe function.

Signed-off-by: Stefan Bader <stefan.bader@canonical.com>
Cc: Alan Cox <alan@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/kobil_sct.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/kobil_sct.c
+++ b/drivers/usb/serial/kobil_sct.c
@@ -114,6 +114,7 @@ static struct usb_serial_driver kobil_de
 	.usb_driver = 		&kobil_driver,
 	.id_table =		id_table,
 	.num_interrupt_in =	NUM_DONT_CARE,
+	.num_interrupt_out = 	NUM_DONT_CARE,
 	.num_bulk_in =		0,
 	.num_bulk_out =		0,
 	.num_ports =		1,
